docs/docs/developers/create-and-develop-project/diagrams/src/runtime-docker-compose.puml

```diff
@@ -0,0 +1,54 @@
+'Dynamic Solutions WebSight
+'Copyright (C) 2013-2022 Dynamic Solutions
+'
+'This is a private, internal code. Using or copying as a
+'whole or in fragments disallowed without permission.
+
+'https://plantuml.com/deployment-diagram
+
+
+@startuml
+!include <tupadr3/devicons/docker>
+
+actor :Developer: as dev
+actor :Content Author: as contentAuthor
+actor :Site Visitor: as siteVisitor
+
+rectangle host as "Host" #line.dotted {
+    rectangle dockerCLI as "Docker CLI"
+    rectangle docker as "Docker Engine" #line.dotted {
+        rectangle dockerDeamon as "Docker Daemon"
+        rectangle  repository as "Docker Repository" #line.dotted {
+            file wscmsImage as "WebSight CMS CE - Project Image - includes application"
+            file nginxImage as "NGINX - Project Image - includes configuration"
+        }
+        frame containers as "Containers" #line.dotted {
+            node nginx as "<$docker>\n  NGINX + config" #royalblue;text:white
+            node mongo as "<$docker>\n  MongoDB" #royalblue;text:white
+            node wscmsce as "<$docker>\n   WS CMS CE + app" #royalblue;text:white
+        }
+        frame volumes as "Filesystem" #line.dotted {
+            database htmlVolume as "Html storage" #royalblue;text:white
+            database jcrVolume as "Content Repository" #royalblue;text:white
+        }
+    }
+}
+
+cloud dockerHub as "Docker Hub"
```

Comment:
I think it might be removed. It might be too much details to connect it with MongoDB container.